build-ci.rb

```diff
@@ -17,11 +17,21 @@ class Project
 
   DEFAULT_MODE = 'test'.freeze
 
-  def initialize(name)
+  def initialize(name, test_type: :rspec, title: nil, weight:)
     @name = name
+    @title = title || name
+    @test_type = test_type
+    @weight = weight
   end
 
-  ALL = %w[api backend core frontend sample].map(&method(:new)).freeze
+  ALL = [
+    new('api', weight: 50),
+    new('backend', weight: 215),
+    new('backend', test_type: :teaspoon, title: "backend JS", weight: 15),
+    new('core', weight: 220),
+    new('frontend', weight: 95),
+    new('sample', weight: 22)
```

Comment:
curiosity: how did you find/choose these weights?

Reply:
Just basically `time rspec` on my local machine. Not very accurate (my computer is a lot faster than circle), but it just needs to be an estimate